automation/run-tests.sh

```diff
@@ -258,8 +258,11 @@ then
 fi
 
 docker_exec 'while ! systemctl is-active dbus; do sleep 1; done'
+docker_exec 'systemctl start systemd-udevd
+             while ! systemctl is-active systemd-udevd; do sleep 1; done
+'
 docker_exec '
-    systemctl start NetworkManager
+    systemctl restart NetworkManager
```

Comment:
Why is this needed?

Reply:
Based on my tests, the NM service needs to be restarted after systemd-udevd is running, otherwise it does not work. That is why we need to wait for that daemon to be active before we proceed.

Reply:
this would be strange because we did not have to restart NM before and the breaking change was that systemd-udev was not installed anymore.